packages/ember-htmlbars/tests/helpers/view_test.js

```diff
@@ -366,436 +61,99 @@ QUnit.test('Should apply class without condition always', function() {
 
-  var parentView = firstChild(view);
-  var childView  = firstGrandchild(view);
+  run(function() {
+    set(view, 'output', 'you are so <i>super</i>');
+  });
 
-  equal(get(parentView, 'ohai'), childView);
+  equal(view.$().text(), 'you are so <i>super</i>', 'updates with entities, not elements');
+  equal(view.$('i').length, 0, 'does not create an element when value is updated');
 });
 
-QUnit.test('{{view}} id attribute should set id on layer', function() {
-  owner.register('template:foo', compile('{{#view view.idView id="bar"}}baz{{/view}}'));
-
-  var IdView = EmberView;
-
+QUnit.test('should not escape HTML in triple mustaches', function() {
   view = EmberView.create({
-    [OWNER]: owner,
-    idView: IdView,
-    templateName: 'foo'
+    template: compile('{{{view.output}}}'),
+    output: 'you need to be more <b>bold</b>'
   });
 
   runAppend(view);
 
-  equal(view.$('#bar').length, 1, 'adds id attribute to layer');
-  equal(view.$('#bar').text(), 'baz', 'emits content');
-});
+  equal(view.$('b').length, 1, 'creates an element');
 
-QUnit.test('{{view}} tag attribute should set tagName of the view', function() {
-  owner.register('template:foo', compile('{{#view view.tagView tag="span"}}baz{{/view}}'));
+  run(function() {
+    set(view, 'output', 'you are so <i>super</i>');
+  });
 
-  var TagView = EmberView;
+  equal(view.$('i').length, 1, 'creates an element when value is updated');
+});
 
+QUnit.test('should not escape HTML if string is a Handlebars.SafeString', function() {
   view = EmberView.create({
-    [OWNER]: owner,
-    tagView: TagView,
-    templateName: 'foo'
+    template: compile('{{view.output}}'),
+    output: new SafeString('you need to be more <b>bold</b>')
   });
 
   runAppend(view);
 
-  equal(view.$('span').length, 1, 'renders with tag name');
-  equal(view.$('span').text(), 'baz', 'emits content');
-});
+  equal(view.$('b').length, 1, 'creates an element');
 
-QUnit.test('{{view}} class attribute should set class on layer', function() {
-  owner.register('template:foo', compile('{{#view view.idView class="bar"}}baz{{/view}}'));
+  run(function() {
+    set(view, 'output', new SafeString('you are so <i>super</i>'));
+  });
 
-  var IdView = EmberView;
+  equal(view.$('i').length, 1, 'creates an element when value is updated');
+});
 
+QUnit.test('should teardown observers from bound properties on rerender', function() {
   view = EmberView.create({
-    [OWNER]: owner,
-    idView: IdView,
-    templateName: 'foo'
+    template: compile('{{view.foo}}'),
+    foo: 'bar'
   });
 
   runAppend(view);
 
-  equal(view.$('.bar').length, 1, 'adds class attribute to layer');
-  equal(view.$('.bar').text(), 'baz', 'emits content');
+  equal(observersFor(view, 'foo').length, 1);
+
+  run(function() {
+    view.rerender();
+  });
+
+  equal(observersFor(view, 'foo').length, 1);
 });
 
 QUnit.test('{{view}} should not allow attributeBindings to be set', function() {
```

Comment:
Now covered in https://github.com/emberjs/ember.js/pull/13236/files#diff-c47395a5aaf92aa0a7edbfa812d4e0afR373